src/databricks/labs/remorph/config.py

```diff
@@ -51,13 +23,19 @@ class TranspileConfig:
     schema_name: str = "transpiler"
     mode: str = "current"
 
-    def get_read_dialect(self):
-        return get_dialect(self.source_dialect)
+    @property
+    def input_path(self):
+        if self.input_source is None:
```

Comment:
```suggestion
        if not self.input_source:
```
This way we can also check for both empty strings and None